### ISSUE TITLE
Artist image and bio mismatch when mutli value tags enabled

### ISSUE THREAD

**Issue by rfporter** [sentriz/gonic#548]
gonic version: v0.16.4

if from docker, docker tag: sentriz/gonic:v0.16.4
if from source, git tag/branch:

When multi value tag recognition is enabled:
GONIC_MULTI_VALUE_ARTIST = delim ;
GONIC_MULTI_VALUE_ALBUM_ARTIST = delim ;

Multi artist albums are indexed correctly however artist images and bios are no longer matched up to the artist correctly. For example the artist Common Rider may show the image and bio of Alberta Cross. Artist genre seems to be correct though.

The source of the artist bio and image (I'm assuming) is the artist.nfo and folder.jpg in the artist folder.

**Comment by sentriz** [comment]
hi, that's right artist images on the filesystem can't work any more with multi value tags. necessarily, since there is no longer a 1-1 relationship between artist folder and tag

i would recommend setting a lastfm api key in the gonic settings, that way gonic can fetch artist images (and other stuff) from lastfm

**Comment by rfporter** [comment]
Just to be clear, the mismatch between artist and bio/picture happens to all artists, not just artists that have multi-artist albums.

**Comment by sentriz** [comment]
did you update from a previous version to 0.6.x? artist images from filesystem were removed. maybe you still had old ones in the image cache

**Comment by rfporter** [comment]
nope. I didnt upgrade from 0.6.x.

I have deleted the database and and cache, then rescanned. This time with:

GONIC_MULTI_VALUE_ARTIST = multi
GONIC_MULTI_VALUE_ALBUM_ARTIST = multi

(library is tagged using picard id3v2.4 UTF-8)

I get the same result. Artist bio and images dont match up.

I have confirmed though that the images and bio are being pulled from last.fm and not the local nfo and folder.jpg. This is even before setting up the link to last.fm.

After setting up the link with last.fm, I see no change to bio and artist image.

I am using Symfonium as the test client.
